specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-02-02-preview/managedClusters.json

```diff
@@ -4856,6 +4856,9 @@
         },
         "monitoring": {
           "$ref": "#/definitions/NetworkMonitoring"
+        },
+        "advancedNetworking": {
+          "$ref": "#/definitions/AdvancedNetworking"
```

Comment:
Please apply these changes to 2024-0**3**-02-preview/managedClusters.json

Reply:
woops. I have moved it